### ISSUE TITLE
Password-hash Error with DB

### ISSUE THREAD

**Issue by JacobYim** [JacobYim/CSE442-BuyNSell#57]
Even though follow the issue #56, Acceptance test is not passed.
```
Pass test1
Pass test2
Pass test3
Pass test4
Pass test5
Pass test6
Pass test7
Fail test8 : valid signin is failed
Fail test9 : valid signin is failed
Pass test10
```

**Comment by JacobYim** [comment]
It would be problem related with the database.

Because Hash has 56 length, the database password has to be assign more than 56 length.

**Comment by JacobYim** [comment]
@Yenrenkhor suggested me

In database SQL,
```
create table user_profile(
user_id serial primary key,
ubid varchar(12) unique not null,
email varchar(50) unique not null,
username varchar(50) unique not null,
password varchar(50) not null
);
```

should be changed to 
```
create table user_profile(
user_id serial primary key,
ubid varchar(12) unique not null,
email varchar(50) unique not null,
username varchar(50) unique not null,
password varchar(60) not null
);
```

**Comment by JacobYim** [comment]
To test the changes,

```
docker-compose down
docker-compose up -d 
node server.js
```
and run the acceptance test for the signin function at test directory.

```
Junghwan-ui-MacBook-Pro:test junghwanyim$ python3 signin_acceptance_test.py 
Pass test1
Pass test2
Pass test3
Pass test4
Pass test5
Pass test6
Pass test7
Pass test8
Pass test9
Pass test10
```

Passed all test cases.

**Comment by JacobYim** [comment]
This is solved at #58
